Mathlib/Init.lean

```diff
@@ -49,3 +50,25 @@ All linters imported here have no bulk imports;
 - the `upstreamableDecls` linter, as it is also mostly informational
 
 -/
+
+-- Define a linter set of all mathlib syntax linters which are enabled by default.
+-- Projects depending on mathlib can use `set_option linter.allMathlibLinters true` to enable
+-- all these linters. XXX can they also enable them in their lakefile?
+register_linter_set linter.allMathlibLinters :=
```

Comment:
I like to use the word "standard" because it's both the default for Mathlib, and also it means that the linted code is up to Mathlib's standards :)
```suggestion
register_linter_set linter.mathlibStandardSet :=
```

Reply:
I like it!